### ISSUE TITLE
Load custom fonts

### ISSUE THREAD

**Issue by ifearthelord** [moehriegitt/NeatLauncher#10]
I have always wanted a launcher like this and I like it very much.
But there's one thing that can be added to it which will greatly improve the ux- the ability to load custom fonts.

Or if there are problems in adding that feature- such as visual glitches etc. then the least that could be done in that case is to remove the reduntant fonts like 'cursive' and add better ones like "circular" "poppins" "exo2" "fira-code" etc.

I hope you will take this into consideration!
Thanks for the launcher :)

**Comment by moehriegitt** [comment]
Yes, I would like a custom font option, too, but I know of no way how to implement it.  I don't think it's possible to load a font dynamically from the sdcard or something like that. For that reason, the best second thing to do would be a have a good selection of fonts that are built into the package.

I've already compiled a list of a few fonts with a compatible free license that allows packaging the font with an app.  The ones I found, howver, are not really interestingly different from what's already there.

We could use this thread to assembly a list of a few good fonts that are reasonably different and have a free license allowing packaging.

**Comment by moehriegitt** [comment]
'Poppins' as a free download, but I don't find the license for it.

**Comment by ifearthelord** [comment]
Have a look at "circular"  "liberation mono" 
And "Grouptix Flow"  "Groutpix Flow Slab Serif" from here-
https://ffeeaarr.my.id/collection/all
